### PR TITLE
feat(#196) - Remove the tabs from home page and change to linear structure

### PR DESCRIPTION
**Pull request by hybridx**
(feat#196) - Removed from tab and added in the page according to the designs and also updated the test cases

# Closes/Fixes/Resolves

###  resolves#196

# Explain the feature/fix

Implemented according to the new designs provided 
https://xd.adobe.com/view/efe4d47d-9981-422e-4e7f-d19446acdb31-99d5/

## Does this PR introduce a breaking change

No

## Screenshots

Screenshot 2020-04-23 at 9 53 12 AM

Screenshot 2020-04-23 at 10 02 04 AM
Screenshot 2020-04-23 at 10 02 07 AM
Screenshot 2020-04-23 at 10 02 09 AM
Screenshot 2020-04-23 at 10 02 10 AM

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Comment by deshmukhmayur** [comment]
The designs need some modifications with the order of the sections.
When it's done and ack'ed, update the PR. And also, add some margin between the 2 sections. They are too close.

File: packages/home-spa/src/js/controller.js [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
+import '../img/login.svg';
+import '../img/magnifying-glass.svg';
 
 const microserviceCard = document.getElementById('microservice-cards');
-const spaList = document.getElementById('banner__links');
 const carouselSlide = document.getElementById('carousel-slide');
+const spaList = document.getElementById('banner__links');
 
-microserviceCard.innerHTML = `<div class="loader"></div>`;
+microserviceCard.innerHTML = `<div></div><div class="loader"></div>`;
```
Comment (deshmukhmayur) [inline packages/home-spa/src/js/controller.js:15]: Why is there an empty `div` here?

Comment (hybridx) [reply packages/home-spa/src/js/controller.js:15]: To position the loader in the center as its a grid the loader takes up the first gird block

**Comment by hybridx** [comment]
> Also, remove the old screenshot/gif from the PR description. Just keep the one that is accurate.

Removed the gif, and kept the one that shows the margin and also few others

File: packages/home-spa/src/js/controller.js (hunk 1 above)
Comment (deshmukhmayur) [reply packages/home-spa/src/js/controller.js:15]: You can use the grid-column/grid-row (whichever fits best) css property to place the div in the right spot.

Comment (hybridx) [reply packages/home-spa/src/js/controller.js]: Required styles added and extra div removed

**Pull request merged**